src/Database/Statement/StatementDecorator.php

```diff
@@ -39,7 +39,7 @@ class StatementDecorator implements StatementInterface, Countable, IteratorAggre
      * Statement instance implementation, such as PDOStatement
      * or any other custom implementation.
      *
-     * @var \Cake\Database\StatementInterface
+     * @var \PDOStatement
```

Comment:
We should always keep the interface in `|` appended IMO for all cases where we "are forced" to go back to the concrete one.